pyproject.toml

```diff
@@ -31,28 +31,43 @@ dependencies = [
     "xhtml2pdf==0.2.16",
 ]
 
-[tool.isort]
-profile = "black"
-force_single_line = "true"
+[tool.ruff]
+line-length = 120
 
-[tool.flake8]
-max-line-length = 120
-count = true
+target-version = "py310"
+
+[tool.ruff.lint]
+select = [
+    "E",  # pycodestyle
+    "W",  # pycodestyle
+    "F",  # pyflakes
+    "I",  # isort
+    "B",  # flake8-bugbear
+    "C90",  # mccabe cyclomatic complexity
+    "G",  # flake8-logging-format
+]
+ignore = []
+exclude = [
+    "db/migrations/versions/",
```

Comment:
Same comment as on frontend PR, do we want to drop this db migrations exclusion?

_Edit: Just saw your reply to my comment on the other PR so yeah all good, we can tidy this up when we rationalise_